### PR TITLE
Delivery report upload and metrics display on dashboard

### PR DESCRIPTION
**Pull request by IonutCiuta**

### CODE REVIEW

File: src/app/pages/delivery-reports/delivery-reports.component.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          (res) => {
+            this.deliveryReport = res;
+            console.log('Delivery report', this.deliveryReport);
+          },
+          (err) => console.log(err)
+        );
+    }
+
+    public upload(projectId, data) {
+        let uploadURL = `http://localhost:9090/projects/${projectId}/deliveryReports/upload`;
```
Comment (vmoldoveanu) [inline src/app/pages/delivery-reports/delivery-reports.component.ts]: We should create an environment file and compose the URL based on that. Not urgent but will save some time later Ex:
local: 'localhost:9090',
dev:'domain:9090'
etc

File: src/app/pages/delivery-reports/delivery-reports.component.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    border-right: 1px solid rgba(0, 0, 0, 0.12);
+    padding: 0.5rem;
+}
+
+.status {
+    font-size: 20px !important;
```
Comment (vmoldoveanu) [inline src/app/pages/delivery-reports/delivery-reports.component.scss]: !important is the nightmare of styling :) - should be removed

File: src/app/pages/delivery-reports/delivery-reports.component.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                            Density of Duplication
+                        </div>
+                        <div class="col-5 spaced">
+                            {{ deliveryReport.metricsReport.duplicationDensityValue.toFixed(2) }}%
+                        </div>
+                        <div class="col-2 status">
+                            <i [ngClass]="getStatusStyle(deliveryReport.metricsReport.duplicationDensityStatus)"></i>
+                        </div>
+                    </div>
+                    <div class="row p-0" style="border-top: 1px solid rgba(0, 0, 0, 0.12);">
```
Comment (vmoldoveanu) [inline src/app/pages/delivery-reports/delivery-reports.component.html]: no inline styles

File: src/app/pages/delivery-reports/delivery-reports.component.scss (hunk 2 above)
Comment (IonutCiuta) [reply src/app/pages/delivery-reports/delivery-reports.component.scss]: True, missed it

File: src/app/pages/delivery-reports/delivery-reports.component.ts (hunk 1 above)
Comment (IonutCiuta) [reply src/app/pages/delivery-reports/delivery-reports.component.ts]: Moved URL to enviroment file. Also, I moved the excel upload in ProjectService. If all is good, I'll move the other project related API calls there as well.

**Pull request merged**